### DISCUSSION TITLE
Free open source organization account

### DISCUSSION THREAD

**Discussion by SebastienEske** [community/community#23272] [New to GitHub]
<p>Hello,</p>
<p>My organization has an account on some other system than github where we added some features to several github open source projects. The problem: organization accounts are all paying and contributing to open source projects would really be the only reason for us to use github. So either we pay double the price for our repos (obviously we won’t do that) or we just don’t contribute back to the open source community (and we don’t want to do that either).</p>
<p>So would it be possible to have a free open source organization account if all your repos are public ? It doesn’t seem too hard to do and would really help open source projects hosted on github.</p>
<p>Since these repos are already hosted somewhere else, maybe you can also add the ability to add external (to github) repos. They would obviously have a limited functionality (such as just making pull request to other repos on git, the main thing to contribute to open source projects) but then, they would not take any disk space on github servers either. This is more complicated, I know…</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/sebastieneske">@sebastieneske</a>,</p>
<p>Thanks for being here! Team is free for Open Source check out the details: <a href="https://help.github.com/en/articles/types-of-github-accounts#organization-accounts" rel="nofollow noopener">https://help.github.com/en/articles/types-of-github-accounts#organization-accounts</a></p>
<p>I hope this helps!</p>

**Comment by SebastienEske** [comment]
<p>Oh! great!</p>
<p>Then you probably want to add this information here as well: <a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a></p>
<p>Best regards</p>
<p>Sebastien</p>

**Comment by AndreaGriffiths11** [comment]
<p>For sure, thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate team.</p>

**Comment by niolap** [comment]
<p>Hello,</p>
<p>I don’t see any more detail about the free Team for Open Source organization on that link you mentioned.</p>
<p>How can I convert my organization to an Open Source organization to get the plan free Team for Open Source?</p>
<p>Thank you,</p>
<p>Nicolas</p>

**Comment by majormoses** [comment]
<p>I think this process must have changed because there used to be the option when creating the org for open source.</p>
